authentication-typescript/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## v1.3.0 (September 30, 2021)
+
+### Changed
+- Upgrade PX Blue packages
+
```

Comment:
You can consolidate the notes from 1.2.1 under 1.3.0 since 1.2.1 never got published